source/live.html.haml

```diff
@@ -0,0 +1,16 @@
+---
+title: Demuxed 2020
+---
+
+%section.live-embed#js-live-embed
+  // %iframe{:src => "https://live-2020-demuxed-com.vercel.app/"}
+:javascript
+  var section = document.getElementById('js-live-embed');
+  var iframe = document.createElement('iframe');
+  var search = window.location.search || '';
+  // iframe.setAttribute('src', 'https://live-2020-demuxed-com.vercel.app/' + search);
+  iframe.setAttribute('src', 'http://localhost:3000/' + search);
+  iframe.setAttribute('height', window.innerHeight - 200); // subtract 200 for the top nav and the bottom line of the footer
```

Comment:
this makes sense on desktop, but doesn't really make sense on mobile, need to see what to do here